site/en/docs/devtools/console/log/index.md

```diff
@@ -162,16 +138,12 @@ page.
 
     {% Img src="image/admin/YZ9nzR7Gm4xJWkcJM2Jt.png", alt="Enabling the Verbose log level.", width="800", height="518" %}
 
-    **Figure 16**. Enabling the **Verbose** log level.
-
 4.  Click **Cause Violation**. The page becomes unresponsive for a few seconds and then the browser
     logs the message `[Violation] 'click' handler took 3000ms` to the Console. The exact duration
     may vary.
 
     {% Img src="image/admin/9mzXfbfY3s7aFzfUWG4H.png", alt="A violation in the Console.", width="800", height="518" %}
 
-    **Figure 17**. A violation in the Console.
-
 ## Filter messages {: #filter }
 
 On some pages you'll see the Console get flooded with messages. DevTools provides many different
```

Comment:
Somewhere in the doc, we should mention the filter message only work on output message. Any input mesages are not filter. 


For example, type `console.warn('test')` in the console. it will output "test" in yellow background. When filter by error message, only the output line "test" is hide. The input `console.warn('test')` is still available.